### ISSUE TITLE
Touch Bar Server "crashes" touchbar

### ISSUE THREAD

**Issue by alexjshepler** [bikkelbroeders/TouchBarDemoApp#113]
When I launch the touch bar server app my touch bar goes black and doesn't display anything. When I hit fn none of the function keys show up either.

**Comment by minksW** [comment]
yeah, latest update has been since 2015. don't think it's gonna work on this version of macosx.
the least version that works with this is el capitan (10.11)

**Comment by RemoPro** [comment]
No, the Touch Bar only exists since 2016 and macOS 10.12 Sierra

**Comment by minksW** [comment]
okay, well it's just not gonna work on any modern version of macosx. 
mostly because of how old this is, not worth it posting issues about a 6-7 project that most likely has been inactive.
